pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/gates/tests/CMakeLists.txt

```diff
@@ -27,9 +27,12 @@ target_sources(${PL_BACKEND}_gates_tests INTERFACE runner_${PL_BACKEND}_gates.cp
 ################################################################################
 # Define targets
 ################################################################################
-set(TEST_SOURCES    Test_MPSTNCuda_NonParam.cpp
+set(TEST_SOURCES    
+                    Test_MPSTNCuda_NonParam.cpp
                     Test_ExaTNCuda_NonParam.cpp
-                    Test_MPSTNCuda_Param.cpp)
+                    Test_MPSTNCuda_Param.cpp
+                    Test_ExaTNCuda_Param.cpp
+                    )
```

Comment:
When setting test sources we usually start the file names in the first line. Besides the set() call.